android/modules/ui/src/java/ti/modules/titanium/ui/widget/webview/TiWebChromeClient.java

```diff
@@ -87,6 +88,15 @@ public boolean onConsoleMessage(ConsoleMessage message)
 		return true;
 	}
 
+	@Override
+	public void onPermissionRequest(final PermissionRequest request)
+	{
+		TiViewProxy proxy = tiWebView.getProxy();
```

Comment:
Unused?

Reply:
correct, tried a different approach before and that survived the code change. Gone now